gulpfile.js

```diff
@@ -0,0 +1,162 @@
+const gulp = require('gulp');
+const gulpWebpack = require('webpack-stream');
+const webpack = require('webpack');
+const sass = require('gulp-sass');
+const maps = require('gulp-sourcemaps');
+// const minifyCss = require('gulp-minify-css');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const childProcess = require('child_process');
+const angularProtractor = require('gulp-angular-protractor');
+const KarmaServer = require('karma').Server;
+
+var apiFiles = ['./*.js', './lib/*.js', './models/*.js', './routes/*.js'];
+var appFiles = ['./app/**/*.js'];
+var testFiles = ['./test/*test.js'];
+var unitFiles = ['./test/unit/**/*test.js'];
+var specFiles = ['./test/integration/**/*spec.js'];
+
+var children = [];
+
+gulp.task('startservers:test', () => {
+  process.env.PORT = 5505;
+  const mongoURI = 'mongodb://localhost/fit_cliques_testDB';
+  children.push(childProcess.spawn('mongod', ['--dbpath=./db']));
+  children.push(childProcess.fork('server.js', [], {
+    env: {
+      MONGODB_URI: mongoURI,
+      PORT: 5505
+    }
+  }));
+});
+
+gulp.task('webpack:dev', ['html:dev', 'css:dev'], () => {
+  return gulp.src('app/js/entry.js')
+    .pipe(gulpWebpack({
+      devtool: 'source-map',
+      output: {
+        filename: 'bundle.js'
+      },
+      plugins: [
+        new webpack.EnvironmentPlugin(['PORT'])
+      ]
+    }))
+    .pipe(gulp.dest('./build'));
+});
+
+gulp.task('html:dev', () => {
+  return gulp.src('app/**/*.html')
+    .pipe(gulp.dest('/build'));
+});
+
+gulp.task('sass:dev', () => {
+  gulp.src('./app/**/*.scss')
```

Comment:
We could actually just have this point to our main.scss file, and with it's require statements will pull in all the other files. Probably doesn't matter, but is going to be slightly fatster to build.

This is what I did for my gulp task for scss compiling, and it worked just fine.